recipes/default.rb

```diff
@@ -24,33 +24,19 @@
 # WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 #
 
-if node['os_version'] >= "6.2" 
-  features = ["Printing-Server-Foundation-Features", "Printing-Server-Role",]
-  if registry_data_exists?(
-    "HKEY_LOCAL_MACHINE\\Software\\Microsoft\\Windows NT\\CurrentVersion",
-    { :name => "InstallationType", :type => :string, :data => "Server" },
-    :machine
-  )
-    features << "Printing-AdminTools-Collection"
+features = ['Printing-Server-Foundation-Features', 'Printing-Server-Role']
+# Install additional role if Full Server Install (not ServerCore)
+if registry_data_exists?(
+  'HKEY_LOCAL_MACHINE\\Software\\Microsoft\\Windows NT\\CurrentVersion',
+  { name: 'InstallationType', type: :string, data: 'Server' },
+  :machine
+)
+  features << 'Printing-AdminTools-Collection'
+end
+features << 'ServerManager-Core-RSAT'
+features << 'ServerManager-Core-RSAT-Role-Tools'
+features.each do |feature|
+  windows_feature feature do
```

Comment:
windows_feature can take an array, which will run a lot faster. Just build the array and then pass the whole thing in

Reply:
There was a unique case between servercore and server which required an additional feature, but it would be easier to read splitting into two `windows_feature` resources.

Would it be a better presentation to